integration/network/bridge/iptablesdoc/generated/swarm-portmap.md

```diff
@@ -0,0 +1,148 @@
+## Swarm service, with a published port
```

Comment:
Let's do that in a follow-up - perhaps also as part of the nftables effort ... it'll mean a bit of shenanigans in the script that generates the doc that won't really be related to this change.

I added the swarm doc as part of this change to show that its main set of rules isn't affected, because the code behind them has been modified in this PR (and generation of rules in those other netns's hasn't changed).